pkg/endpoint/endpoint.go

```diff
@@ -281,6 +282,8 @@ type Endpoint struct {
 	// realizedBPFConfig is the config currently active in the BPF datapath.
 	realizedBPFConfig *bpfconfig.EndpointConfig
 
+	LXCMap *lxcmap.LXCMap `json:"-"`
```

Comment:
TBH it feels odd to have a global variable "shared" across all entities. This could leave under daemon (which I believe it already does) and when the endpoint would like to access the LXCMap it would do it through the owner interface.

Reply:
Sure, I can do that for now.